scripts/src/main/resources/scripts/functions

```diff
@@ -836,7 +836,7 @@ function doReplaceExtractedSkipSingleFolder() {
   if [ "${#*}" = 1 ] && [ "${1%.app}" = "${1}" ] && [ -d "${1}" ]
   then
     files="$(find "${dir}" -maxdepth 1 -mindepth 1 -exec basename {} \;)"
-    anz_files="$(echo "${files}" | wc -w)"
```

Comment:
It seems that the wc implementation on some MacOS systems produces leading spaces. 
Example: https://github.com/alfeilex/ide/actions/runs/4108659467/jobs/7089608184#step:5:436

I have found some solutions to fix this. I chose the `awk` option, but you can see other alternatives here:
https://unix.stackexchange.com/questions/205906/extra-space-with-counted-line-number/205910#205910

Reply:
Thanks for the explanation.
Indeed we should avoid calling external processes like `find`, `awk` or `wc` at all for such things, but I will create another PR for this. I resolve this conversation to allow merging.